src/App.tsx

```diff
@@ -1,24 +1,125 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { createTodo, getTodos, deleteTodo } from './api/todos';
+import { Footer } from './components/footer_filter/footer_filter';
+import { Header } from './components/header_input/header_input';
+import { Main } from './components/main_todos-list/main_todos-list';
 
-const USER_ID = 0;
+const USER_ID = 10548;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [input, setInput] = useState('');
+  const [status, setStatus] = useState('all');
+  const [addTodo, setAddTodo] = useState('');
+  const [removeTodo, setRemoveTodo] = useState(0);
```

Comment:
looks like it is redundant states
```suggestion
```
instead create state for tempTodo
![Screenshot from 2023-05-27 17-50-30](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/33029864/1af426de-8b82-47b6-85c9-e7ce52b69989)